xgi/classes/reportviews.py

```diff
@@ -688,3 +692,55 @@ def singletons(self):
 
         """
         return self.filterby("size", 1)
+
+    def maximal(self):
+        """Returns the maximal edges as an EdgeView
```

Comment:
I'd make it more explicit what a maximal edge, for example:
```suggestion
        """Returns the maximal edges as an EdgeView. Maximal edges are those that are not subsets of any other edges in the hypergraph.
```